src/Stratis.Bitcoin/BlockPulling/BlockPuller.cs

```diff
@@ -244,6 +253,35 @@ public void Initialize(OnBlockDownloadedCallback callback)
             this.logger.LogTrace("(-)");
         }
 
+        /// <inheritdoc />
+        public void RequestWitnessOptions()
```

Comment:
here we might want to add a param for NetworkPeerServices to be able to request different options than `NODE_WITNESS` but I'm not sure if we really need that so this method is for witness only right now.

Reply:
Ah yes I agree

Reply:
fixed